README.md

```diff
@@ -23,16 +23,16 @@ import Numerics
 ```
 
 Swift Numerics modules have minimal dependencies on other projects.
-The current modules assume only the availability of the Swift and C standard libraries, and the runtime support provided by compiler-rt.
-Future expansion may assume the availability of other standard interfaces such as BLAS and LAPACK, but modules with more specialized dependencies (or dependencies that are not available on all platforms supported by Swift) belong in a separate package.
+The current modules assume only the availability of the Swift and C standard libraries and the runtime support provided by compiler-rt.
+Future expansion may assume the availability of other standard interfaces such as [BLAS (Basic Linear Algebra Subprograms)](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms) and [LAPACK (Linear Algebra Package)](https://en.wikipedia.org/wiki/LAPACK), but modules with more specialized dependencies (or dependencies that are not available on all platforms supported by Swift) belong in a separate package.
 
-Because our intention is to make it possible to adopt Swift Numerics modules in the standard library at some future point, Swift Numerics uses the same license and contribution guidelines as the Swift project.
+Because we intend to make it possible to adopt Swift Numerics modules in the standard library at some future point, Swift Numerics uses the same license and contribution guidelines as the Swift project.
 
 ## Process
 Swift Numerics is a standalone library separate from the core Swift project.
-In practice it will act as a staging ground for some APIs that may eventually be incorporated into the Swift Standard Library, and when that happens such changes will be proposed to the Swift Standard Library using the established evolution process of the Swift project.
+In practice, it will act as a staging ground for some APIs that may eventually be incorporated into the Swift Standard Library, and when that happens such changes will be proposed to the Swift Standard Library using the established evolution process of the Swift project.
 
-Swift Numerics uses github issues to track bugs and features, and pull requests for development.
+Swift Numerics uses GitHub issues to track bugs and features and pull requests for development.
```

Comment:
```suggestion
Swift Numerics uses GitHub issues to track bugs and features, and pull requests for development.
```

Reply:
+1

Reply:
But _", and"_ is grammatically incorrect... Do you like this, it is both grammatically correct and easy to read:
### Alternative 1 - Ampersand
_"Swift Numerics uses GitHub issues to track bugs & features and pull requests for development."_

### Alternative 2 - Slash:
_"Swift Numerics uses GitHub issues to track bugs/features and pull requests for development."_

### Alternative 3 - "or":
_"Swift Numerics uses GitHub issues to track bugs or features and pull requests for development."_

🤔

Reply:
Those are all bad.

*Swift Numerics uses GitHub issues to track bugs and features. We use pull requests for development.*

Reply:
@stephentyrone great, fixed! Thanks!

Reply:
> But ", and" is grammatically incorrect...

FWIW, it is grammatically just fine here, whatever the tools try to tell you.

Reply:
@xwu It is the case of `compound predicate`, see screenshot:

<img width="977" alt="Screenshot 2019-11-08 at 17 05 25" src="https://user-images.githubusercontent.com/864410/68490915-23d1aa00-024a-11ea-91ed-d466edb7c51f.png">

Reply:
There's no second verb. Even in the case of a compound predicate, it's fine to use a comma when it's necessary for other reasons. You can ignore the tool.